Resfeber/View Controllers/ExploreViewController.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  ExploreViewController.swift
+//  Resfeber
+//
+//  Created by David Wright on 11/8/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+class ExploreViewController: UIViewController {
+    
+    // MARK: - Properties
+    
+    fileprivate var collectionView: UICollectionView!
+    
+    fileprivate let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 28, weight: .bold)
+        label.textColor = .label
+        label.text = "Explore"
+        return label
+    }()
+    
+    fileprivate let profileButton: UIButton = {
+        let button = UIButton(type: .system)
+        let buttonDiameter: CGFloat = 32
+        button.setDimensions(height: buttonDiameter, width: buttonDiameter)
+        button.layer.cornerRadius = buttonDiameter / 2
+        button.layer.masksToBounds = true
+        button.contentMode = .scaleAspectFill
+        button.backgroundColor = .systemGray3
+        button.addTarget(self, action: #selector(profileImageTapped), for: .touchUpInside)
+        
+        let image = UIImage(systemName: "person.fill")!.withTintColor(.systemGray6, renderingMode: .alwaysOriginal)
+        button.setImage(image, for: .normal)
```

Comment:
`button.setImage()` accepts an optional UIImage, so you probably don't need to force unwrap on line 35.

Reply:
Agreed, I will optional chain it instead